htdocs/projet/tasks.php

```diff
@@ -171,6 +173,7 @@
 	't.progress_summary' => array('label' => "TaskProgressSummary", 'checked' => 1, 'position' => 10),
 	't.budget_amount' => array('label' => "Budget", 'checked' => 0, 'position' => 11),
 	'c.assigned' => array('label' => "TaskRessourceLinks", 'checked' => 1, 'position' => 12),
+	't.billable' => array('label' => "Billable", 'checked' => 1, 'position' => 13),
```

Comment:
This property must be moved 2 lines after, inside the test 
`if ($object->usage_bill_time) {`
So it will appear only if the project has the usage "Bill tasks"

Reply:
Still ok, pushing change.